runners/core-java/src/main/java/org/apache/beam/runners/core/TimerInternals.java

```diff
@@ -242,6 +242,7 @@ public int compareTo(TimerData that) {
       }
       ComparisonChain chain =
           ComparisonChain.start()
+              .compare(this.getOutputTimestamp(), that.getOutputTimestamp())
```

Comment:
nit: this looks unrelated

Reply:
Yes, I'm removing it, also missed that is useless as that comparison is made later anyway